src/test/java/za/ac/cput/repository/LecturerRepositoryImplTest.java

```diff
@@ -9,8 +9,13 @@
 import static org.junit.jupiter.api.Assertions.*;
 @TestMethodOrder(MethodOrderer.MethodName.class)
 class LecturerRepositoryImplTest {
+<<<<<<< HEAD
```

Comment:
I see that you had conflict issues that you did not resolve before sending a pull request. 
Please resolve them so that the file is free of foreign characters.
`<<<<<< HEAD`
`>>>>>>> cc76966ecbbc0f8c68525a302105f231ee648e71`